aspnetcore/fundamentals/static-files/samples/1x/StartupAddHeader.cs

```diff
@@ -13,15 +13,16 @@ public void ConfigureServices(IServiceCollection services)
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         #region snippet_ConfigureMethod
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+            var cachePeriod = env.IsDevelopment() ? 600 : 604800;
```

Comment:
These should be strings.

Reply:
At least if they're numbers they validate (as numbers of seconds) on build. Magic strings ... I wouldn't have gone that way in my own hacks.